src/models/subscription.ts

```diff
@@ -190,7 +190,7 @@ export default class Subscription extends Sequelize.Model {
 	static async startNewSyncProcess(subscription: Subscription) {
 		//TODO Add a sync start logic
 		await sqsQueues.backfill.sendMessage({
-			gitHubInstallationId: subscription.gitHubInstallationId,
+			installationId: subscription.gitHubInstallationId,
```

Comment:
this could be confusing as we do have an 'Installation' model with an id

Reply:
This is our current payload that we store in Redis. We have to have the same for an easy migration, unless I miss something..

Reply:
Plz check http://github.com/atlassian/github-for-jira/blob/a807b2195065a48f68a89ec3f37a09ec3bc2fa3e/src/sync/installation.ts#L454-L454

Reply:
ah right, we really should fix that at some point before someone mistaken it